### PR TITLE
CB-2801: extend subnet range

### PR DESCRIPTION
**Pull request by tiborpopovics**

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java [hunk 1]
```diff
@@ -55,6 +60,10 @@ private void validateNetwork(EnvironmentCreationDto request, String cloudPlatfor
         }
     }
 
+    private boolean isInvalidNetworkMask(String networkCidr) {
+        return !networkCidr.split("/")[1].equals(EXPECTED_NETWORK_MASK);
```
Comment (keyki) [inline environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java]: I think we should check if the cidr actually contains the subnet mask and it's not just an IP address.

Comment (tiborpopovics) [reply environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java]: Good idea. Please check now.

Comment (bergerdenes) [reply environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java]: there is a validator for this, take a look on that code ;)

Comment (tiborpopovics) [reply environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java]: I`ve updated wrong validation. Check it again please.

Comment (keyki) [reply environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java]: Can you please use the SubnetValidator?

**Pull request merged**